### PR TITLE
Popover

### PR DESCRIPTION
**Pull request by tassoevan**

### CODE REVIEW

**Comment by ggazzo** [comment]
This pull request **fixes 1 alert** when merging 7762381ff2f85e2e06441837aeed8162671ffa2f into d85e1a651f3258e2f73d1828b4abb237b487e049 - [view on LGTM.com](https://lgtm.com/projects/g/RocketChat/Rocket.Chat.Livechat/rev/pr-3e70d925f94c58857de782e6628b7e85201616a6)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by tassoevan** [comment]
@renatobecker You need to wrap the components around a `PopoverContainer`, like we do with the `StoreProvider`.

**Comment by renatobecker** [comment]
@tassoevan, yeah, sorry.. It was missing the `PopoverContainer's' stuff.
I fixed this after commenting here but I forgot to remove the comment.

Thanks.

File: src/components/Popover/index.js [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+	state = {
+		renderer: null,
+	}
+
+	open = (renderer, props, { currentTarget } = {}) => {
+		let overlayBounds;
+		let triggerBounds;
+
+		if (this.overlayRef) {
+			overlayBounds = normalizeDOMRect(this.overlayRef.getDOMNode().getBoundingClientRect());
```
Comment (renatobecker) [inline src/components/Popover/index.js]: @tassoevan, an error is being raised at this point:

screen shot 2019-01-01 at 16 01 05

**Comment by ggazzo** [comment]
This pull request **fixes 1 alert** when merging ef554e0d6f5bf71a65e383eb69d0a610b0aedcb6 into 61da1a442c3bb7cf1f638eb8da94aaf10c2bd30e - [view on LGTM.com](https://lgtm.com/projects/g/RocketChat/Rocket.Chat.Livechat/rev/pr-a33fdab7cec7097f9cdd862fcee0986ac181d2ab)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by ggazzo** [comment]
This pull request **fixes 1 alert** when merging 07e607b060a051859c7fcda97089539d10394ec3 into 2f4db8ae4554b6b23927b883c5c73ce8aef8d92c - [view on LGTM.com](https://lgtm.com/projects/g/RocketChat/Rocket.Chat.Livechat/rev/pr-0b959c6d5e89e9956a6b6cddb3d5e24ef455af2b)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by ggazzo** [comment]
This pull request **fixes 1 alert** when merging 9e63913d2fbcabc98a30cab392bc17b10f8c55e1 into 2f4db8ae4554b6b23927b883c5c73ce8aef8d92c - [view on LGTM.com](https://lgtm.com/projects/g/RocketChat/Rocket.Chat.Livechat/rev/pr-2e4c81dc46b31cba6936c28fe48dc7388a087269)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by ggazzo** [comment]
This pull request **fixes 1 alert** when merging 7dc20deb1f23ad168c32236258b109187e8ad3eb into 2f4db8ae4554b6b23927b883c5c73ce8aef8d92c - [view on LGTM.com](https://lgtm.com/projects/g/RocketChat/Rocket.Chat.Livechat/rev/pr-d9c9ded98b262150da5b3c6f14b777a2aad14aae)

**fixed alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**